instrumentation/opentelemetry-instrumentation-aws-sdk/src/AwsSdkInstrumentation.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use Opentelemetry\API\Common\Instrumentation\InstrumentationInterface;
+use Opentelemetry\API\Common\Instrumentation\InstrumentationTrait;
+
+
+class AwsSdkInstrumentation implements InstrumentationInteface
+{
+
+    use InstrumentationTrait;
+
+    private $instrumented;
+
+    public function getName(): string
+    {
+        return 'AWS SDK Instrumentation';
+    }
+
+    public function getVersion(): ?string
+    {
+        return '0.0.1';
+    }
+
+    public function getSchemaUrl(): ?string
+    {
+        return true;
```

Comment:
This should return the url of the used Otel semantic convention schema, or in case if it's unkown `null`